03_hardware_configuration/slides/slides.md

```diff
@@ -0,0 +1,631 @@
+</header>
+
+  <div class="flex-col justify-center">
+    <p>To operate a system, you obviously need not only the PLC but also the corresponding modules, motors, and so on. Accordingly, AX also offers various options for incorporating configuration information for modules into the project. There are two options here:</p>
+        <ul>
+            <li>Importing the hardware as a standardized GSDML format.</li>
+            <li>Using standard Siemens modules that are supported and provided as apax packages that can be included as dev dependencies.</li>
+        </ul>
+    <br>
+    <p>In the following section we will show you how to proceed in order to have the hardware information available in the project.</p>
+    <br>
+    <p>We will show how the modules are integrated into the project in the following chapter on templates.</p>
+  </div>
+
+----
+
+<header class="slide_header">
+    <h3>Use of GSD Files</h3>
+</header>
+
+<div class="flex-col justify-center">
+
+  <p>The GSD (General Station Description) format is a description language used to describe and parameterize hardware settings. It is a standard supported by all PROFINET device manufacturers. Accordingly, each hardware manufacturer provides the description file.
+  <br>
+  <p>First, you need to contact the manufacturer of the module you want to use and obtain the GSD file. This is then usually inserted into the project. 
+    We recommend creating a folder called `gsd` in the project and inserting the files there centrally. After that, you need to make the module known in the project so that you can use it. This is done with the help of the hardware compiler. You call:</p>
+  <br>
+  <p><b>apax hwc install-gsd --input &lt;name_of_GsdFile&gt;</b></p>
+  <br>
+  <p>The install-gsd command will generate intermediate files that are required for Hardware Compiler from the GSDML or GSDX files.</p>
+  <br>
+  <p>These intermediate files will be placed in the GSD installation cache folder. If you use Hardware Compiler version 3.1 or later, it is located in the .ax folder of your user folder. After this, the hardware compiler is ready to use the information from the GSDML file by the compile.</p>
+  <br>
+  <ol>
+    <li>Additional information about GSDML: <a href="https://www.profibus.de/downloads/gsdml-gsdx-specification-for-profinet">GSDML Specification for PROFINET</a></li>
+    <li>Additional information about the import of GSDML Files: <a href="https://console.simatic-ax.siemens.io/docs/hw/language/gsd-devices-and-modules">GSDML information on AX Documentation</a></li>
+  </ol>
+</div>
+
+----
+
+<header class="slide_header">
+    <h3>Extension GSDML Viewer</h3>
+</header>
+<div class="grid-two-col-eq">
+    <div class="flex-col justify-center">
+       <p>Depending on how many assemblies it describes, a GSD file can become very large, making it difficult to determine which attributes an assembly has and which values it can assume.</p>
```

Comment:
```suggestion
       <p>Depending on how many devices and modules it describes, a GSD file can become very large, making it difficult to determine which configuration parameters an module has and which values it can assume.</p>
```